app/include/zmk/ble.h

```diff
@@ -41,6 +47,9 @@ int zmk_ble_unpair_all(void);
 
 int zmk_ble_set_device_name(char *name);
 
+void zmk_ble_adv_mode_set(bool mode);
+bool zmk_ble_adv_mode_get(void);
```

Comment:
"mode" but using a boolean here feels off. Either return a struct for some mode , or use "adv_enabled" wording, IMHO.

Reply:
The functions are now called `zmk_ble_adv_enabled_set(bool adv_enabled)` and `zmk_ble_adv_enabled_get()`